package.json

```diff
@@ -1,42 +1,51 @@
 {
   "name": "guides-app",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "private": true,
-  "description": "Small description for guides-app goes here",
+  "description": "Ember app that powers https://guides.emberjs.com",
   "license": "MIT",
   "author": "Chris Manson <chris@stonecircle.io>",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
-  "repository": "https://github.com/stonecircle/guides-app",
+  "repository": "https://github.com/ember-learn/guides-app",
   "scripts": {
     "build": "ember build",
     "build-production": "ember build -e production",
-    "lint:js": "eslint ./*.js app config lib server tests",
+    "lint:js": "eslint ./*.js app blueprints config lib server tests",
     "start": "ember serve",
     "test": "ember test"
   },
-  "devDependencies": {
-    "@ember/guides-source": "github:ember-learn/guides-source#c18b8c5c3d9a404f509d64c187ff36e87b9ae374",
+  "dependencies": {
+    "@ember/guides-source": "github:ember-learn/guides-source#v3.1.4",
```

Comment:
why is the version locked down here? I'm curious to know what our deployment patterns should be as we move forward (not today)

Reply:
This is a github release tag. Referring to commit ids can make it hard to go back and track that commit inside the source repo compared to release tags.

On deployments,
I'm hoping that whenever we create github releases/tags on guides-source, then may be via travis we can raise a PR for this repo.